deploy/runtools/runtime_config.py

```diff
@@ -181,13 +201,18 @@ def array_to_lognames(values: Sequence[Optional[Any]], prefix: str) -> List[str]
 
         screen_name = "fsim{}".format(slotid)
         run_device_placement = "+slotid={}".format(slotid)
-        # TODO: re-enable for vitis
-        #other = "+binary_file={}".format(self.xclbin) if self.platform == "vitis" else ""
+
+        if self.platform == "vitis":
+            assert self.xclbin is not None
+            vitis_bit = "+binary_file={}".format(self.xclbin)
```

Comment:
This is really tricky to remove... worst case this is not dealt with in this release but in the "cleanup release"